test/errors/duplicateEnumElement-detailed.good

```diff
@@ -0,0 +1,30 @@
+─── error in duplicateEnumElement.chpl:1 [MultipleEnumElems] ───
+  Enum 'languagesILike' has multiple elements named 'chapel'.
+      |
+    1 | enum languagesILike {
+    2 |   chapel, cpp, haskell,
+    3 |   chapel, chapel
+    4 | }
+      |
+  Referenced here:
+      |
+    5 | var chpl = languagesILike.chapel;
```

Comment:
Is the implication that if the duplicate symbol weren't referenced, we wouldn't generate an error?  If so, that seems problematic (in that there's a bug in the enum just waiting for someone to trip on it).  If not, then I'd just remove this section that talks about where it's referenced since it's not relevant (which I might do in either case).

Reply:
What would it take to move this duplicate detection at the point of the enum's definition instead?  It seems as thought it could essentially be done at parse-time, e.g. (not that I'm suggesting that).

Reply:
My first thought was that it can be a parse check, actually. I'd need to compare against what we currently do and don't check at parse time to confirm that this is a good idea.

Reply:
To clarify my "I'm not suggesting that" comment:  It'd be fine with me if it were done at parse-time, assuming that's not counter to dyno philosophy (some might argue that parsing should just change AST into IR and that any semantic checks should be done separately/later, but if dyno doesn't stick to that approach, that's fine with me.  My uncertainty is why I shied away from suggesting it.